package/test/test.js

```diff
@@ -1,20 +1,99 @@
-import { spawn } from "child_process";
-import _VAPI from "../dist/index.js";
+import Ajv from "ajv";
```

Comment:
Why is this file tracked via git?

Reply:
Its the tests for the api

Reply:
It tests each method on the class and compares its returned data to the expected return data structure using JSON schema + AJV. The `testOptions.ts` has the parameters and expected return type for each method

Reply:
Oh, I see. Are you asking why the js files are tracked by git? It was just to eliminate having to recompile them, but I'll make a script to compile them